specification/contosowidgetmanager/Contoso.Management/employee.tsp

```diff
@@ -20,9 +20,6 @@ model EmployeeProperties {
   /** Age of employee */
   age?: int32;
 
-  /** City of employee */
-  city?: string;
-
   /** Profile of employee */
```

Comment:
```suggestion
  /** City of employee */
  city?: string;

  /** Profile of employee */
```

Reply:
Link to the apiview diff of this deletion: https://apiviewstagingtest.com/Assemblies/Review/0fb8a03ff8c348308ed5dc7cce897b08?revisionId=73515ea0b20d4a568e3b5c642e89eba3&diffOnly=False&doc=False&diffRevisionId=853cb125e21f4c4fa9e02d1e72fe0c98